lib/Test/Mojo.pm

```diff
@@ -191,31 +191,40 @@ sub header_unlike {
     $regex, $desc);
 }
 
+sub _does_json_contain {
+  my ($self, $p) = @_;
+  Mojo::JSON::Pointer->new($self->tx->res->json)->contains($p);
```

Comment:
This method is not in the right place in the file. All methods are ordered alphabetically. I also think I would rename it so it takes less time to parse what the method does:

```perl
sub _json_contains {
  return Mojo::JSON::Pointer->new(shift->tx->res->json)->contains(shift);
}
```